invenio_records_rest/facets.py

```diff
@@ -34,14 +34,25 @@
 
 
 def terms_filter(field):
-    """Create a term filter."""
+    """Create a term filter.
+
+    :param field: field name.
```

Comment:
Can you please check that you are consistently using capital letter at the beginning and dot at the end of each sentence?